examples/train_unconditional.py

```diff
@@ -34,29 +36,37 @@ def main(args):
         logging_dir=logging_dir,
     )
 
-    model = UNet2DModel(
-        sample_size=args.resolution,
-        in_channels=3,
-        out_channels=3,
-        layers_per_block=2,
-        block_out_channels=(128, 128, 256, 256, 512, 512),
-        down_block_types=(
-            "DownBlock2D",
-            "DownBlock2D",
-            "DownBlock2D",
-            "DownBlock2D",
-            "AttnDownBlock2D",
-            "DownBlock2D",
-        ),
-        up_block_types=(
-            "UpBlock2D",
-            "AttnUpBlock2D",
-            "UpBlock2D",
-            "UpBlock2D",
-            "UpBlock2D",
-            "UpBlock2D",
-        ),
-    )
+
+    model_dir = f"{args.output_dir}/unet"
+    if os.path.exists(model_dir):
+        print(f"reloading model from {model_dir}")
+        model = UNet2DModel.from_pretrained(model_dir)
+        print(f"reloaded model from {model_dir}")
```

Comment:
Instead of print statements it's better to use `logger.info(...)` to make the amount of logging controllable by the user.